sdk/communication/Azure.Communication.Common/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fixed the logic of `PhoneNumberIdentifier` to always maintain the original phone number string whether it included the leading `+` sign or not.
```

Comment:
it should be a patch release so pls change the version to 1.2.1

Reply:
But I can see this unreleased version was set already. Do you want me to additionally add another unreleased version information in the changelog?

Reply:
you can change it to the one suggested by Aigerim + also in the .csproj file
once released, a new version moniker will be added to both files